examples/plot_simulate_alpha.py

```diff
@@ -43,8 +42,7 @@
 location = 'proximal'
 burst_std = 20
 weights_ampa_p = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
-syn_delays_p = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
-                'L5_basket': 1., 'L5_pyramidal': 1.}
+syn_delays_p = {'L2_pyramidal': 0.1, 'L5_pyramidal': 1.}
```

Comment:
Figured out why this was giving an error for me earlier (messed up variables when testing in jupyter notebook). Much more clear now